### PR TITLE
Add note on Dapr and Contour

### LINKED ISSUE #372: "Why does Radius install Dapr by default?"
We had several test users who were aware of Dapr wonder why it's installed with Radius. 

They noticed it in the example output to verify the Radius control-plane was deployed: 
image
(https://radapp.dev/getting-started/tutorial/init-environment/#initialize-a-radius-environment)

**Proposed ideas**
- We could add a note about this to our FAQ page at https://radapp.dev/reference/faq/
- We could build out a page that gives a description of each of the control plane services and link out to it from that part of the tutorial. e.g. in the Tutorial add "For an explanation of what each of these services are, here's a link to that page."
  - One usability tester suggested we also add notes about what it's doing in the cluster generally, e.g. any namespace we're creating.

### PR DESCRIPTION
**Pull request by AaronCrawfis**
Closes #372

### CODE REVIEW

**Review by emily-potyraj — APPROVED** [review]
🚀 LGTM!

File: docs/content/getting-started/tutorial/1-tutorial-environment/index.md [hunk 1]
```diff
@@ -69,6 +69,9 @@ In this step we will be initializing a Radius Kubernetes environment.
    dapr-sentry               1/1     1            1           35s
    dapr-operator             1/1     1            1           35s
    ```
+
+   Note that Dapr and Contour are also both installed into the `radius-system` namespace to help get you up and running quickly for our quickstarts and tutorial. This is a [point-in-time limitation]({{< ref limitations >}}) that will be addressed with richer environment customization in a future update.
```
Comment (DarqueWarrior) [inline docs/content/getting-started/tutorial/1-tutorial-environment/index.md:73]: Should it be tutorials with an "s" or singular as it is? I think with an s like on qucikstarts...

File: docs/content/reference/limitations.md [hunk 2]
```diff
@@ -6,6 +6,14 @@ description: "Learn where there are known issues and limitations with the latest
 weight: 998
 ---
 
+## Radius control plane
+
+### `rad install kubernetes` and `rad env init kubernetes` installs Dapr and Contour in addition to Radius
+
+Dapr and Contour are also both installed into the `radius-system` namespace to help get you up and running quickly for our quickstarts and tutorial. This is a point-in-time limitation that will be addressed with richer environment customization in a future update.
```
Comment (DarqueWarrior) [inline docs/content/reference/limitations.md:13]: Same feedback as above. Might need an s on tutorial

**Review by DarqueWarrior — DISMISSED** [review]

File: docs/content/getting-started/tutorial/1-tutorial-environment/index.md (hunk 1 above)
Comment (AaronCrawfis) [reply docs/content/getting-started/tutorial/1-tutorial-environment/index.md:73]: We switched to a singular tutorial for Project Radius a couple months ago, with multiple quickstarts.

**Pull request merged**